var/spack/repos/builtin/packages/metabat/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Metabat(SConsPackage):
+    """MetaBAT, an efficient tool for accurately reconstructing single
+    genomes from complex microbial communities."""
+
+    homepage = "https://bitbucket.org/berkeleylab/metabat"
+    url      = "https://bitbucket.org/berkeleylab/metabat/get/v2.12.1.tar.gz"
+
+    version('2.12.1', 'c032f47a8b24e58a5a9fefe52cb6e0f8')
+
+    depends_on('boost@1.55.0:', type=('build', 'run'))
+
+    def setup_environment(self, spack_env, run_env):
+        spack_env.set('BOOST_ROOT', self.spec['boost'].prefix)
```

Comment:
I'm not familiar with this particular package, but most SCons packages I've seen clean the environment and ignore environment variables unless you explicitly pass them in. Is this not the case here?

Reply:
As you noted, the Scontruct file contains `ENV_update(env['ENV'], os.environ)` allowing the BOOST_ROOT to be set with setup_environment.